guides/v2.2/javascript-dev-guide/javascript/js_mixins.md

```diff
@@ -133,6 +156,22 @@ var config = {
 };
 ```
 
+The following is an example of a `requirejs-config.js` file that adds the `step-navigator-mixin` mixin defined in the previous example, to the [step navigator object][].
+
+**File:** `OrangeCompany/Sample/view/frontend/requirejs-config.js`
+
+```javascript
+var config = {
```

Comment:
Please, merge this example with the previous one. That's fair that it's applicable only for the frontend area. However, multiplying similar examples might be an overhead for the reader. So, in this particular case, we can move this declaration to the base area (just add the declaration to the previous example)

Reply:
Fixed it, thank you!